src/Button/IconicButton.story.js

```diff
@@ -65,6 +66,16 @@ WithACustomSize.story = {
   name: "with a custom size",
 };
 
+export const WithACustomColor = () => (
+  <IconicButton color={theme.colors.red} icon="close" labelHidden>
```

Comment:
You don't need to pull the theme in here. Since storybook is wrapped in the `NDSProvider`, theme values can be used directly, e.g `<IconicButton color="red" />`.